components/CheckBox/CheckBox.type.tsx

```diff
@@ -0,0 +1,14 @@
+export interface ICheckboxProps {
+  isChecked: boolean
+  isDisabled: boolean
+  isReadonly: boolean
+  label: string | undefined
+  id: string | undefined
+  className?: string | undefined
+  name: string | undefined
+  direction?: 'flex | flex-col'
```

Comment:
direction?: 'flex-row' | 'flex-col'
each of items must be in inside " ", and use flex-row